### PR TITLE
CTCP-4994: update validation on unlocode to accept lowercase/uppercase

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: app/controllers/loading/UnLocodeController.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ class UnLocodeController @Inject() (
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, departureId, mode))),
           value =>
-            service.doesUnLocodeExist(value).flatMap {
+            service.doesUnLocodeExist(value.toUpperCase).flatMap {
```
Comment (jackheslop96) [inline app/controllers/loading/UnLocodeController.scala:71]: Consider handling this in the form provider with
`"value" -> adaptedText(s"$prefix.error.required")(_.toUpperCase)`

File: app/controllers/locationOfGoods/UnLocodeController.scala [hunk 2]
```diff
@@ -66,7 +66,7 @@ class UnLocodeController @Inject() (
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, departureId, mode))),
           value =>
-            service.doesUnLocodeExist(value).flatMap {
+            service.doesUnLocodeExist(value.toUpperCase).flatMap {
```
Comment (jackheslop96) [inline app/controllers/locationOfGoods/UnLocodeController.scala:69]: As above

**Comment by kenneth-rayner** [comment]
test this please

**Review by jackheslop96 — APPROVED** [review]

File: conf/logback.xml [hunk 3]
```diff
@@ -8,18 +8,6 @@
         </encoder>
     </appender>
 
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%date{ISO8601} level=[%level] logger=[%logger] thread=[%thread] rid=[%X{X-Request-ID}] user=[%X{Authorization}] message=[%message] %replace(exception=[%xException]){'^exception=\[\]$',''}%n</pattern>
-        </encoder>
-    </appender>
```
Comment (jackheslop96) [inline conf/logback.xml:15]: Did you mean to remove this one?

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**